ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+        systemctl daemon-reload 
+        systemctl enable $1.service 
+        systemctl start $1.service 
+        if [ "$?" != 0 ]; then
+            echo "${1^} could not be started."
+            exit 1;
+        else
+            echo "${1^} started"
+        fi
+    elif [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+        chkconfig $1 on 
+        service $1 start 
+        /etc/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            echo "${1^} could not be started."
+            exit 1;
+        else
+            echo "${1^} started"
+        fi
+    elif [ -x /etc/rc.d/init.d/$1 ] ; then
+        /etc/rc.d/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            echo "${1^} could not be started."
+            exit 1;
+        else
+            echo "${1^} started"
+        fi
+    else
+        echo "Error: ${1^} could not start. No service manager found on the system."
+        exit 1;
+    fi
+}
+
+## Show script usage
+getHelp() {
+
+   echo ""
+   echo "Usage: $0 arguments"
+   echo -e "\t-d   | --debug Shows the complete installation output"
+   echo -e "\t-i   | --ignore-health-check Ignores the health-check"
+   echo -e "\t-h   | --help Shows help"
+   exit $1 # Exit script after printing help
+}
+
+
+## Install the required packages for the installation
+installPrerequisites() {
+    logger "Installing all necessary utilities for the installation..."
+    $sys_type install curl unzip wget libcap -y -q 
+    echo -e '[AdoptOpenJDK] \nname=AdoptOpenJDK \nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public' | tee /etc/yum.repos.d/adoptopenjdk.repo $debug
+    $sys_type install adoptopenjdk-11-hotspot -y -q 
+    export JAVA_HOME=/usr/
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Prerequisites could not be installed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Add the Wazuh repository
+addWazuhrepo() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Adding the Wazuh repository..."
+    rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH 
+    if [ "${STATUS_PACKAGES}" = "prod" ]; then
+      logger "Adding production repository..."
+      echo -e "[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/${WAZUH_MAJOR}.x/yum/\nprotect=1" | tee /etc/yum.repos.d/wazuh.repo $debug
+    elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+      logger "Adding development repository..."
+      echo -e '[wazuh_pre-release]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/pre-release/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo $debug
+    fi
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Wazuh repository could not be added"
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2